9c-internal/terraform/main.tf

```diff
@@ -0,0 +1,72 @@
+provider "aws" {
+  region  = "us-east-2"
+  profile = "planetarium-dev"
+}
+
+terraform {
+  required_providers {
+    aws = {
+      version = ">= 3"
+    }
+  }
+  required_version = ">= 0.13.0"
```

Comment:
this would be better to be on terraform block in
`backend.tf`
```
terraform {
  required_providers {
    aws = {
      version = ">= 3"
    }
  }
  required_version = ">= 0.13.0"
  
  backend "s3" {
    bucket = "terraform-eks-backend"
    key    = "eks/9c-vpc"
    region = "us-east-2"
  }
}
```